packages/react-core/src/components/Slider/examples/Slider.md

```diff
@@ -134,13 +135,22 @@ class ContinuousInput extends React.Component {
     return (
       <>
         <Text component={TextVariants.h3}>Slider value is: {this.state.value.toFixed(2)}</Text>
-        <Slider value={this.state.value} onChange={this.onChange} />
+        <Checkbox
+          label="thumbHasTooltip"
+          isChecked={this.state.thumbHasTooltip}
+          onChange={thumbHasTooltip => this.setState({ thumbHasTooltip })} />
+        <Text component={TextVariants.h3}>Slider Value is: {this.state.value.toFixed(2)}</Text>
+        <Slider
+          thumbHasTooltip={this.state.thumbHasTooltip}
+          value={this.state.value}
+          onChange={this.onChange} />
         <br />
         <Text component={TextVariants.h3}>Slider value is: {this.state.valueCustom.toFixed(2)}</Text>
```

Comment:
As it is, the `Slide value is:` text in the example only has 2 values trailing the decimal, but the tooltip has many. I wonder if we should make the number of trailing values something the user can configure, and have it always default to 2? It'd make the aria label much more readable.

Reply:
Made it default to 2 decimals. Lets add the property for configuring once someone requests it if you 're fine with that.

Reply:
Could you also remove the trimming in the examples if it's no longer needed?

Reply:
Done